lib/rubocop/cop/style/send.rb

```diff
@@ -0,0 +1,19 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for the use of the send method.
+      class Send < Cop
+        MSG = 'Prefer `BasicObject#__send__` or `Object#public_send` to `send`,
+               as `send` may overlap with existing methods.'
+
+        def on_send(node)
+          _receiver, method_name, *_args = *node
+          return unless method_name == :send
```

Comment:
looking at the `debugger` cop, I think the method_name check should be sufficient.